resources/views/auth/info.blade.php

```diff
@@ -0,0 +1,159 @@
+@extends('layouts.app')
+
+@section('content')
+    <div class="container">
+        <div class="row">
+            <div class="col-md-10 col-md-offset-1">
+                <div class="card card-default">
+                    <div class="card-header">@lang('user.personal_information'):</div>
+                    <div class="card-body">
+                        <form method="post" action="">
+                            @csrf
+                            <div class="form-group">
+                                <label class="col-sm-4 control" class="col-sm-4 control" for="name">@lang('user.name')</label>
+                                <div class="col-sm-8">
+                                <input class="form-control" type="text" name="name" id="name" required>
+                                </div>
+                            </div>
+
+                            <div class="form-group">
+                                <label class="col-sm-4 control" for="surname">@lang('user.surname')</label>
+                                <div class="col-sm-8">
+                                <input class="form-control" type="text" name="surname" id="surname" required>
+                                </div>
+                            </div>
+                            <div class="form-group">
+                                <label class="col-sm-4 control" for="email">@lang('user.email')</label>
+                                <div class="col-sm-8">
+                                <input class="form-control" type="email" name="email" id="email" required>
+                                </div>
+                            </div>
+                            <div class="form-group">
+                                <label class="col-sm-4 control" for="date_and_place_of_birth">@lang('user.date_and_place_of_birth')</label>
+                                <div class="col-sm-8">
+                                <input class="form-control" class="control" type="text" name="date_and_place_of_birth"
+                                       id="date_and_place_of_birth" required>
+                                </div>
+                            </div>
+                            <div class="form-group">
+                                <label class="col-sm-4 control" for="mother_name">@lang('user.mother_name')</label>
+                                <div class="col-sm-8">
+                                <input class="form-control" type="text" name="mother_name" id="mother_name" required>
+                                </div>
+                            </div>
+                            <div class="form-group">
+                                <label class="col-sm-4 control" for="phone_number">@lang('user.phone_number')</label>
+                                <div class="col-sm-8">
+                                <input class="form-control" type="text" name="phone_number" id="phone_number" required>
```

Comment:
Since regexp pattern validation is hard fro phone numbers (*international stuffs*), I would simply add a placeholder to ensure at least some consistency regards to phone number formats.

```html
<input class="form-control" type="text" name="phone_number" id="phone_number" placeholder="+36 12 345 6789" required>
```